tools/go_generics/globals/globals_visitor.go

```diff
@@ -47,6 +47,11 @@ type globalsVisitor struct {
 
 	// scope is the current scope as nodes are visited.
 	scope *scope
+
+	// processAnon indicates whether we should process anonymous struct fields.
+	// does not make strict checking on parameter types that share the same name
```

Comment:
Please make this a full sentence.

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences